backend/dissemination/views.py

```diff
@@ -244,13 +248,13 @@ def get_audit_content(self, report_id):
 
         # QuerySet values to an array of dicts
         data["Awards"] = [x for x in awards.values()]
-        if notes_to_sefa.exists():
+        if notes_to_sefa.exists() and include_private_and_public:
             data["Notes to SEFA"] = [x for x in notes_to_sefa.values()]
         if audit_findings.exists():
             data["Audit Findings"] = [x for x in audit_findings.values()]
-        if audit_findings_text.exists():
+        if audit_findings_text.exists() and include_private_and_public:
             data["Audit Findings Text"] = [x for x in audit_findings_text.values()]
-        if corrective_action_plan.exists():
+        if corrective_action_plan.exists() and include_private_and_public:
             data["Corrective Action Plan"] = [
                 x for x in corrective_action_plan.values()
             ]
```

Comment:
the `include_private_and_public` checks here will result in each of these rendering with a count of 0 when the user lacks `read-tribal` permissions. Wondering if we need to differentiate between the actual 0 case (i.e. you're allowed to see the count and the count is 0) and the omission case (i.e. you're not allowed to see the count, which may or may not actually be 0). Put another way, is it a problem that in some cases the 0 shown here when the user lacks permission may incidentally be the actual value we're trying to hide? Do we need to show something like N/A when the user lacks permissions?

Reply:
To your question, `Is it a problem that in some cases the 0 shown here, when the user lacks permission, may incidentally be the actual value we're trying to hide?` my answer is no. The only time a report would have 0 findings and 0 captext is when there are no findings associated with the report. However, the findings workbook is public for all reports (including private reports). That is, even if we display N/A instead of 0, people can still tell when 0 findings and 0 captext are expected just by seeing the findings workbook. Regarding `Note` records, this workbook is required for all reports and cannot be submitted empty (i.e., an empty note workbook will not pass validation). So, 0 notes is not a normal case. That said, using N/A instead of 0 is not a heavy lift, so if N/A is preferable here, I can make the update.

Reply:
Made the change to display N/A instead of 0 for private counts.